apis/v1/binding_types.go

```diff
@@ -9,72 +9,79 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// CRPTrackingLabel is the label that points to the cluster resource policy that creates a resource binding.
+const CRPTrackingLabel = labelPrefix + "parentCRP"
+
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:scope=Cluster,categories={fleet},shortName=mc
+// +kubebuilder:resource:scope=Cluster,categories={fleet},shortName=rb
 // +kubebuilder:subresource:status
-// +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="Joined")].status`,name="Joined",type=string
+// +kubebuilder:printcolumn:JSONPath=`.status.conditions[?(@.type=="ResourceBindingApplied")].status`,name="Applied",type=string
 // +kubebuilder:printcolumn:JSONPath=`.metadata.creationTimestamp`,name="Age",type=date
 
-// ResourceBinding is a resource created in the hub cluster to represent a member cluster within a fleet.
-type ResourceBinding struct {
+// ClusterResourceBinding is represents a scheduling decision that binds a group of resources to a cluster.
+// it must have CRPTrackingLabel that points to the cluster resource policy that creates it.
+type ClusterResourceBinding struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
-	// The desired state of MemberCluster.
+	// The desired state of ClusterResourceBinding.
 	// +required
 	Spec ResourceBindingSpec `json:"spec"`
 
-	// The observed status of MemberCluster.
+	// The observed status of ClusterResourceBinding.
 	// +optional
 	Status ResourceBindingStatus `json:"status,omitempty"`
 }
 
-// ResourceBindingSpec defines the desired state of ResourceBinding.
+// ResourceBindingSpec defines the desired state of ClusterResourceBinding.
 type ResourceBindingSpec struct {
-	// ResourceSnapshotNames is the names of the resource snapshot to which
-	// that this resource binding points.
+	// ResourcePolicyName is the name of the resource policy that this resource binding points to.
+	ResourcePolicyName string `json:"resourcePolicyName"`
+
+	// ResourceSnapshotIndex is the index of the resource snapshot to which
+	// that this resource binding points to.
 	ResourceSnapshotIndex int32 `json:"resourceSnapshotIndex"`
```

Comment:
is ResourcePolicyName the policy snapshot name?

Should we only need the ResourceSnapshotName (parent/active)?
The dispatcher will fetch all the resourceSnapshots owned by the same parent and create the corresponding work.